autofac-tests/Ping/IPingService.cs

```diff
@@ -0,0 +1,7 @@
+namespace autofac_tests
+{
+    public interface IPingService
```

Comment:
IMHO: It does not looks like service. ServiceStatusProvider is more suitable name as for me